tests/test_fields.py

```diff
@@ -80,7 +80,7 @@ class DateTimeFieldFormatTest(unittest.TestCase):
     """
     Tests  for `DateTimeFieldFormat`.
     """
-    def test_valid_dates(self):
+    def test_can_valid_dates(self):
         field_format = fields.DateTimeFieldFormat("x", False, None, "YYYY-MM-DD", _ANY_FORMAT)
         field_format.validated("2000-01-01")
```

Comment:
We should compare the result of `validated()` with something specific, not just exercise it. Apparently this test never really was very ambitious. Nevertheless now would be a good time to fix this.

Something like this should work:

```
self.assertEqual('2000-01-01', field_format.validated('2000-01-01').isoformat())
```